frontend/src/json_server/db.json

```diff
@@ -0,0 +1,7476 @@
+{
+  "staff": [
+    {
+      "id": 1,
+      "first_name": "Joe",
+      "surname": "Bloggs",
+      "start_date": "2018-08-03",
+      "contracted_hours": 40,
+      "hourly_rate": 1.23,
+      "role": "USER",
+      "job_title": "barista",
+      "inactive": false
+    },
+    {
+      "id": 2,
+      "first_name": "Mary",
+      "surname": "McGee",
+      "start_date": "2017-08-12",
+      "contracted_hours": 40,
+      "hourly_rate": 999.99,
+      "role": "MANAGER",
+      "job_title": "supervisor",
+      "inactive": false
+    },
+    {
+      "id": 3,
+      "first_name": "Json",
+      "surname": "Server",
+      "start_date": "2015-01-12",
+      "contracted_hours": 40,
+      "hourly_rate": 0.01,
+      "role": "USER",
+      "job_title": "floor",
+      "inactive": false
+    },
+    {
+      "id": 4,
+      "first_name": "Arthur",
+      "surname": "Guinness",
+      "start_date": "2019-12-12",
+      "contracted_hours": 40,
+      "hourly_rate": 2.99,
+      "role": "USER",
+      "job_title": "chef",
+      "inactive": false
+    },
+    {
+      "first_name": "Johnny",
+      "surname": "Herbert",
+      "start_date": "2018-08-03",
+      "contracted_hours": 40,
+      "hourly_rate": 2,
+      "role": "USER",
+      "job_title": "barista",
+      "inactive": false,
+      "id": 5
+    },
+    {
+      "first_name": "Johnny",
+      "surname": "Herbert",
+      "start_date": "2018-08-03",
+      "contracted_hours": 40,
+      "hourly_rate": 2,
+      "role": "USER",
+      "job_title": "barista",
+      "inactive": false,
+      "id": 6
```

Comment:
I don't suppose we need a metric ton of Johnny Herberts?

Reply:
no, we don't, my mistake. I had a test running and it kept adding the same person to the db, and I pushed while the server was running.
But I cannot edit this branch,
"You can’t commit to feature/48-json-server-setup because it is a protected branch."
What should I do?

Reply:
Can you run us through what commands you're using when creating and pushing your branch? It shouldn't be protected by default.

Reply:
ok, that solved it. Thanks